cli.js

```diff
@@ -5,6 +5,7 @@ const InitCommand = require('./cli-commands/init');
 const TestCommand = require('./cli-commands/test');
 const DeployCommand = require('./cli-commands/deploy');
 const CompileCommand = require('./cli-commands/compile');
+const ShapeCommand = require('./cli-commands/shape');
```

Comment:
move it between other rows above so the final view will be ordered by lenght
...
.....
.......
..........